.github/workflows/typing.yml

```diff
@@ -28,9 +28,12 @@ jobs:
       run: |
         pip install .
     - name: Install dependencies
+    # type checking requires protbuf stubs
       run: |
         python -m pip install --upgrade pip
         python -m pip install -r types-requirements.txt
+        python -m pip install grpcio-tools
```

Comment:
Shouldn't grpcio-tools be added to dev_requirements.txt or types-requirements.txt?

Reply:
It's sort of used in both during building / installation, since we need to create `_schema_pb2.py` and `_schema_pb2.pyi`, so not sure :shrug: